keyboards/s7_elephant/rev2/config.h

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2020 MudkipMao
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xA13B // Hexidecimal A13B will be Jacky's vendor ID
+#define PRODUCT_ID      0x0008 // 8 for S7++
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    Jacky
+#define PRODUCT         Jacky S7 Elephant Rev 2
+#define DESCRIPTION     A high-end 70% custom keyboard
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ *
+ *                        0   1   2   3   4   5   6   7   8   9   A   B   C   D   E   F   */
+#define MATRIX_ROW_PINS { B0, B1, B2, B3, B7 }
+#define MATRIX_COL_PINS { D0, D1, D2, D3, D5, D4, D6, D7, B4, F7, F6, F5, F4, F1, F0, E6 }
+
```

Comment:
you should put the matrix size here, 5x16

Reply:
I have it in the config.h in the s7_elephant/ directory. However, I am going to refactor it to model after the kbd67 like you suggested last night